CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- N/A
+### Added
```

Comment:
can you move this to [2.5.0]  so we can release the changes after PR is merged?

Reply:
That's normally done right at the time of release because you don't know the date of the release otherwise.